twint/storage/db.py

```diff
@@ -265,6 +266,7 @@ def tweets(conn, Tweet, config):
                     ",".join(Tweet.cashtags),
                     ",".join(Tweet.urls),
                     ",".join(Tweet.photos),
+                    ",".join(Tweet.thumbnail)
```

Comment:
Here is missing a `,` at the end of the row

Reply:
And below at `cursor.execute` is missing a `,?` too, since there's a new field

Reply:
Embarrassing! it wouldn't require `join()` either since its not a list. Good catch.